### ISSUE TITLE
Navigation by “Tab” doesn’t work in PMC window

### ISSUE THREAD

**Issue by nkolev92** [NuGet/Home#8118]
NuGet Version: NuGet Client 5.0.0.5371

VS Version: D16.0stg 27930.3001

OS Version: Windows 10 RS4_Release 17134.1.180410-1804

 

Installation steps: 

1.     Install VS: \\cpvsbuild\drops\VS\D16.0stg\products\27930.3001\bootstrappers\Enterprise\vs_enterprise.exe

2.     Install NuGet Client: \\ddfiles\Drops\NuGet\Drops\CI\NuGet.Client\dev\5.0.0.5371\artifacts\VS15\NuGet.Tools.vsix

 

Repro steps: 

1. Launch VS and create a C# Class Library (.NET Framework).

2. Navigate to the menu Tools->NuGet Package Manager->Package Manager Console to open the PMC.

3. Tab through the PMC window.

 

Expected:

Navigation by “Tab” works well.

 

Actual:

Navigation by “Tab” doesn’t work in PMC window as attached screenshot (no rectangle show when hitting ‘Tab’).

 

Note:

1. It doesn’t repro on PM UI.

2. It repro on D16.0stg 27930.3001 (default NuGet Client 4.8.0.5278) and D15.9stg 27930.3001 (default NuGet Client 4.8.0.5348).
